examples/with-mongodb/pages/api/hello.js

```diff
@@ -0,0 +1,12 @@
+// Next.js API route support: https://nextjs.org/docs/api-routes/introduction
+
+import { connectToDatabase } from '../../util/mongodb'
+
+export default async (req, res) => {
+  let db = await connectToDatabase()
+
+  let data = await db.collection('test').stats()
+
+  res.statusCode = 200
+  res.json(data)
+}
```

Comment:
Let's use this or something similar for the index page using `getStaticProps`, currently the index page doesn't use MongoDB at all.